lib/iex/resources/symbols.rb

```diff
@@ -0,0 +1,17 @@
+module IEX
+  module Resources
+    class Symbols < Resource
+      property 'symbol'
+      property 'exchange'
+      property 'name'
+      property 'date'
+      property 'is_enabled', from: 'isEnabled'
+      property 'type'
```

Comment:
this field returns a shortcut. I think it would be useful to have a property with "full type".

Reply:
I think it can be done as a separate property. I mean the resource will have both `type` and `full_type` (or `type_descritpion` or something else)

Reply:
Thank you for your input.😁

But since it returns the result from the API, I don't think you need to define it yourself.

If you follow it, you'll have to follow the difference every time.

I feel like that kind of thing is not a debt for this library.

Reply:
What are these types and full types? You can add a `full_type` if it's helpful to the consumer, especially if all consumers of the client need this. I do it all over the place in other libraries ([example](https://github.com/dblock/strava-ruby-client/blob/f439e888130ad809a5680bf8d597e685be597a33/lib/strava/models/mixins/distance.rb#L4)) .